compiler/modules/front/stmt.bal

```diff
@@ -50,6 +55,7 @@ type LoopContext record {|
     // following block is reachable
     Assignment[] onBreakAssignments = [];
     Assignment[] onContinueAssignments = [];
+    LoopKind kind;
```

Comment:
I think it's easier to have `boolean continueIsBackward;`. Then you don't need the LoopKind enum.